portfolio/src/main/webapp/index.html

```diff
@@ -39,20 +41,27 @@ <h1 style="background-color:DodgerBlue;">Kya's Portfolio</h1>
     In my free time I enjoy spending time with friends and going 
     to sporting events. 
 </p>
+
+<form action="/form-handler" method="POST">
+             <p>If interested, please enter your contact information here:</p>
+         <textarea name="text-input"> </textarea>
+        <br/>
+        <input type="submit" />
+        </form>
    
       <img src="/images/headshot.jpg" width="500" height="500" />
       
       
       <img src="/images/Rotunda.jpg" width="500" height="400"/>
       <img src="/images/uva.jpg" width="500" height="400"/>
 
-      <p>Click here to see content created by server code:</p>
-      <button onclink="showHelloWorld()">Hello</button>
-      <div id="my-container"></div>
-      
-      <p>Click here to find out random facts about me:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Click <a href="/hello">here</a> or the button below to see content created by server code. </p>
+      <button onclick="showtheMessage();">Show Message</button>
```

Comment:
is there suppose to be a ; inside here?

Reply:
Yes. They had a semicolon there in the walkthrough example so I'm assuming I should have one here as well